eth/v1alpha1/validator.proto

```diff
@@ -149,19 +149,21 @@ message Validator {
     bool slashed = 4;
 
     // Epoch when the validator became eligible for activation. This field may
-    // be zero if the validator was present in the Ethereum 2.0 genesis.
+    // be zero if the validator was present in the Ethereum 2.0 genesis. This
+    // field is FAR_FUTURE_EPOCH if the validator has not been activated.
     uint64 activation_eligibility_epoch = 5;
 
     // Epoch when the validator was activated. This field may be zero if the
-    // validator was present in the Ethereum 2.0 genesis.
+    // validator was present in the Ethereum 2.0 genesis. This field is
+    // FAR_FUTURE_EPOCH if the validator has not been activated.
     uint64 activation_epoch = 6;
 
-    // Epoch when the validator was exited. This field may be zero if the
-    // validator has not exited.
+    // Epoch when the validator was exited. This field is FAR_FUTURE_EPOCH if
```

Comment:
Do we define FAR_FUTURE_EPOCH anywhere here?

Reply:
I don't think so

Reply:
probably should, other wise could be confusing for someone hasn't seen beacon chain spec

Reply:
Any suggestion where to put it? In the README or ?

Reply:
We should add a link to the spec config variables in the comments of this endpoint as this is a one-off case